metro-ai-suite/metro-vision-ai-app-recipe/loitering-detection/benchmark_app_payload.json

```diff
@@ -0,0 +1,100 @@
+[
+    {
+        "pipeline": "object_tracking_cpu",
+        "payload":{
+            "source": {
+                "uri": "file:///home/pipeline-server/videos/VIRAT_S_000101_looped.mp4",
+                "type": "uri"
+            },
+            "destination": {
+                "metadata": {
+                    "type": "mqtt",
+                    "topic": "object_tracking_$x",
+                    "publish_frame":false
+                },
+                "frame": {
+                    "type": "webrtc",
+                    "peer-id": "object_tracking_$x"
+                }
+            },
+            "parameters": {
+                "detection-properties": {
+                    "model": "/home/pipeline-server/models/intel/pedestrian-and-vehicle-detector-adas-0001/FP16/pedestrian-and-vehicle-detector-adas-0001.xml",
+                    "device": "CPU",
+                    "inference-interval": 3,
+                    "inference-region": 0,
+                    "pre-process-backend": "opencv",
+                    "threshold": 0.7,
+                    "model-instance-id": "instcpu0"
+                }
+            }
+        }
+    },
+    {
+        "pipeline": "object_tracking_gpu",
+        "payload":{
+            "source": {
+                "uri": "file:///home/pipeline-server/videos/VIRAT_S_000101_looped.mp4",
+                "type": "uri"
+            },
+            "destination": {
+                "metadata": {
+                    "type": "mqtt",
+                    "topic": "object_tracking_$x",
+                    "publish_frame":false
+                },
+                "frame": {
+                    "type": "webrtc",
+                    "peer-id": "object_tracking_$x"
+                }
+            },
+            "parameters": {
+                "detection-properties": {
+                    "model": "/home/pipeline-server/models/intel/pedestrian-and-vehicle-detector-adas-0001/FP16/pedestrian-and-vehicle-detector-adas-0001.xml",
+                    "device": "GPU",
+                    "inference-interval": 3,
+                    "inference-region": 0,
+                    "batch-size": 8,
+                    "nireq": 2,
+                    "ie-config": "NUM_STREAMS=2",
+                    "pre-process-backend": "va-surface-sharing",
+                    "threshold": 0.7,
+                    "model-instance-id": "instgpu0"
```

Comment:
Line 55-62 should be the pipeline defaults. Do not overwrite in multiple places.